tests/Elasticsearch/Tests/ClientTest.php

```diff
@@ -88,136 +94,148 @@ public function testFromConfigBadParamQuiet()
             'imNotReal' => 5
         ];
         $client = ClientBuilder::fromConfig($params, true);
+
+        $this->assertInstanceOf(Client::class, $client);
     }
 
-    public function testNullDelete()
+    public function testIndexCannotBeNullForDelete()
```

Comment:
Here I split the original methods with several try/catch blocks into distinct methods with `$this->expectException` and `$this->expectExceptionMessage`